ios/help/start/installing-system-keyboard.md

```diff
@@ -33,9 +33,6 @@ the default iOS keyboard layout by touching the globe key of the keyboard whenev
 ![](../ios_images/globe.png)
 
 ### On "Allow Full Access"
-After opening the Keyman system keyboard for the first time in an app, you can turn off the "Allow Full Access" option
-again.
+For normal use of Keyman, there is no need to keep "Allow Full Access" enabled. It is needed only for making changes to some keyboard preferences.
 
-You may need to temporarily switch this on again in the future for app updates and for some changes to keyboard preferences.
-
-It may always be safely deactivated immediately after opening the system keyboard again in an app.
+However, an issue with iOS 16 causes the Keyman system keyboard to not display unless "Allow Full Access" is enabled. Until the issue is addressed, this setting must be left in the on position for Keyman to operate normally.
```

Comment:
I would advise leading with the later paragraph, then state that "users on versions of iOS before iOS 16.0 do not need to keep..."

Reply:
Good idea -- I wasn't thinking about how the large percentage of users who own devices that will not allow them to upgrade to 16.0. I've revised the text to reflect that.

Reply:
Looks good now. 👍